app/hypnos/config/bootloader.conf

```diff
@@ -9,3 +9,36 @@ CONFIG_FLASH=y
 
 # Memory protection
 CONFIG_MPU_ALLOW_FLASH_WRITE=y
+
+# Allow for large Bluetooth data packets.
+CONFIG_BT_L2CAP_TX_MTU=260
```

Comment:
How did you choose this value ?

Reply:
I simply copied it from the Zephyr SMP sample application. I've not tried excluding it.